hw03_NatsarS.py

```diff
@@ -0,0 +1,65 @@
+#################################################################################
+# Author: Salaam Natsar
+# Username: NatsarS
+#
+# Assignment: Homework 03
+# Purpose: Created a sun flower
+# Google Doc Link: https://docs.google.com/document/d/1gRRxdAQvTMhQ6pwa0BwnBMu09rZ2cglwKLnd5j2zvpg/edit?usp=sharing
+#################################################################################
+# Acknowledgements:
+# https://stackoverflow.com/questions/32804572/how-to-hide-a-turtle-icon-pointer-in-python
+#
+#################################################################################
+
+import turtle
+sunflower = turtle.Turtle()
+sunflower.speed(0)
+s=turtle.Screen()
+s.bgcolor('light green')
+
+def main():
+        """
+        Create the center bulb to our flower!
+        """
+flowercenter = turtle.Turtle()
+flowercenter.hideturtle()
+flowercenter.speed(0)
+flowercenter.goto(-5,-65)
+flowercenter.color("#503D02")
+flowercenter.begin_fill()
+flowercenter.circle(50)  # draws a circle of radius 20
+flowercenter.end_fill()
```

Comment:
This code is not inside `main`; it probably should be. It definitely should not be at this indentation level!